com.raggesilver.BlackBox.json

```diff
@@ -0,0 +1,78 @@
+{
+  "app-id" : "com.raggesilver.BlackBox",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "42",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "blackbox",
+  "finish-args" : [
+    "--allow=devel",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=org.freedesktop.Flatpak"
```

Comment:
why to you need this dbus permission?

Reply:
Black Box needs to call `flatpak-spawn`, which requires the Flatpak dbus.